templates/base.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en-GB" xml:lang="en-GB">
```

Comment:
I believe it's considered [good practice](https://www.w3.org/TR/WCAG20-TECHS/H57.html)